demo2/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,12 @@
+package org.example;
```

Comment:
Use .gitignore to prevent checking in unnecessary files like those in target/ directory